aw_server/log.py

```diff
@@ -1,24 +1,16 @@
 import logging
 
 from werkzeug import serving
-from werkzeug._internal import _log
-
-
-def config_flask_logging():
-    # Fix this
-    # app.logger.setFormatter(logging.Formatter("%(message)s"))
-    pass
 
 
 class FlaskLogHandler(serving.WSGIRequestHandler):
     def __init__(self, *args):
         self.logger = logging.getLogger("flask")
-        self.logger.setLevel(logging.INFO)
```

Comment:
If i remember correctly this was set because flask was spamming the logs even though --testing wasn't specified. Consider putting it back.